### DISCUSSION TITLE
unable to verify student id because of sticker at the back

### DISCUSSION THREAD

**Discussion by Moerven** [community/community#139572] [GitHub Education]
### Hi everyone,

I do own a valid school id but the authentication requires a photo of my id. I can only upload one at the time and i am not allowed to fuse them together. The validation that i am still a student is at the back of the card with a paper sticker. So now i don't know how i can get my education license renewed. I did not have seen any manual support options.

**Comment by VadymMakohon** [comment]
Hello.
Try to reach out to your school and asked for additional document that can be used for verification or maybe try to renew the existing document.
Hope it helps.

**Comment by Kane0023** [comment]
This GitHub discussion addresses an issue with repeated Windows 8.1 corruption. [DJ Aerodynamix](http://djaerodynamix.ca/) users share their experiences and solutions for resolving system crashes, boot loops, and corrupted files on their devices.
